dockerfiles/platform-test/Dockerfile

```diff
@@ -0,0 +1,63 @@
+##############################################################################
+#
+#    HPCC SYSTEMS software Copyright (C) 2020 HPCC Systems®.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+##############################################################################
+
+# Create base container image to be used by all HPCC processes
+# MORE - some of these dependencies are probably not needed by all derived containers - perhaps we should move them
+# Others may not be wanted at all in container mode - scikit example??
+
+ARG BUILD_LABEL
+ARG DOCKER_REPO
```

Comment:
Some of these Args should have sane defaults. We can override DOCKER_REPO but it likely will be hpccsystems in most cases.